Vonk.Plugin.DocumentOperation/DocumentService.cs

```diff
@@ -89,30 +86,41 @@ public async Task Document(IVonkContext vonkContext, string compositionID)
             // Build empty document bundle
             var documentBundle = CreateEmptyBundle();
 
+            vonkContext.Arguments.Handled(); // Signal to Vonk -> Mark arguments as "done"
```

Comment:
Are you sure you want *all* arguments marked as handled?
Or only those you have handled in this plugin?